lib/msal-browser/src/operatingcontext/MetaOSOperatingContext.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { BaseOperatingContext } from "./BaseOperatingContext";
+
+export class MetaOSOperatingContext extends BaseOperatingContext {
+    
+    /*
+     * TODO: Once we have determine the bundling code return here to specify the name of the bundle
+     * containing the implementation for this operating context
+     */
+    static readonly MODULE_NAME: string = "";
+
+    /**
+     * Unique identifier for the operating context
+     */
+    static readonly ID: string = "MetaOSOperatingContext";
```

Comment:
Should we create an enum for these IDs?

Reply:
I don't really want to create an enum of these identifiers, what would I use them for?

Reply:
Don't feel strongly, just a suggestion. If it's not useful we can resolve.